clinics/creator/creator.go

```diff
@@ -17,7 +18,7 @@ const (
 )
 
 type Config struct {
-	ClinicDemoPatientUserId string `envconfig:"CLINIC_DEMO_PATIENT_USER_ID"`
+	ClinicDemoPatientUserId string `envconfig:"DEMO_CLINIC_USER_ID"`
```

Comment:
Where is this configured? I could not find it (or the old name) anywhere in our GH org. Do we need a coordinated set of corresponding config change PRs to change this?